src/Database/Relations/BelongsToMany.php

```diff
@@ -100,6 +100,110 @@ public function create(array $attributes = [], array $pivotData = [], $sessionKe
         return $model;
     }
 
+    /**
```

Comment:
Please provide method summaries for both methods added in their docblocks

Reply:
Sorry, I don't understand you.

Reply:
```php
/** 
  * Summary of what method does goes here
  * 
  * @param type $var blah blah blah
  * @return void
  */
```

Reply:
fixed